axioms_and_computation.md

```diff
@@ -1100,10 +1100,11 @@ theorem linv_comp_self {f : α → β} [Inhabited α]
     have ex  : ∃ a₁ : α, f a₁ = f a := ⟨a, rfl⟩
     have feq : f (choose ex) = f a  := choose_spec ex
     calc linv f (f a)
-      _ = choose ex := dif_pos ex
+      _ = choose ex := by simp
```

Comment:
This fixes an error where `dif_pos`'s implicit arguments were not being found (namely the `Decidable` instance and the else branch). This lemma is really designed to be used with `simp` rather than directly, and `simp` solves the goal.